pkg/kubelet/kubelet.go

```diff
@@ -60,6 +60,7 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/util/bandwidth"
 	utilErrors "k8s.io/kubernetes/pkg/util/errors"
+	kubeioutil "k8s.io/kubernetes/pkg/util/io"
```

Comment:
@thockin @fgrzadkowski whatever alias we choose should be consistent, `ioutil` is used elsewhere.